interfaces/Portalicious/src/app/pages/project/project-team/add-user-button/add-user-button.component.html

```diff
@@ -1,7 +1,94 @@
 <p-button
-  label="Add user to team"
-  i18n-label
-  rounded
+  label="Add team member"
+  i18n-label="@@add-team-member"
+  outlined
   icon="pi pi-plus"
-  (click)="onAdd()"
+  class="text-xs sm:text-base"
+  (click)="formVisible.set(true)"
 />
+
+<app-form-sidebar
+  [(visible)]="formVisible"
+  [formGroup]="formGroup"
+  [mutation]="assignUserMutation"
+  formTitle="Add team member"
+  i18n-formTitle="@@add-team-member"
+>
+  <app-form-field-wrapper
+    label="Choose user"
+    i18n-label
+    [errorMessage]="formFieldErrors()('userValue')"
+    class="mb-3"
+  >
+    <p-dropdown
+      [options]="unassignedUsers()"
+      [optionLabel]="'username'"
+      [optionValue]="'id'"
+      [filter]="true"
+      formControlName="userValue"
+      placeholder="Type or select"
+      i18n-placeholder
+      class="block w-full"
+      styleClass="w-full"
+      [loading]="allUsers.isPending() || !projectUsers"
+    >
+    </p-dropdown>
+  </app-form-field-wrapper>
+
+  <app-form-field-wrapper
+    label="Assign roles"
+    i18n-label
+    [errorMessage]="formFieldErrors()('rolesValue')"
+    class="mb-3"
+  >
+    <p-multiSelect
+      [options]="roles!.data()"
+      [optionLabel]="'label'"
+      [optionValue]="'role'"
+      formControlName="rolesValue"
+      placeholder="Type or select"
+      i18n-placeholder
+      class="block w-full"
+      styleClass="w-full"
+      [panelStyleClass]="
+        'max-w-full [&_.p-multiselect-item]:items-start [&_.p-multiselect-item]:text-wrap'
+      "
+      [showToggleAll]="false"
+      [overlayOptions]="{
+        target: '@parent',
+      }"
+      [loading]="roles.isPending()"
+    >
+    </p-multiSelect>
+  </app-form-field-wrapper>
+
+  @if (enableScope()) {
+    <app-form-field-wrapper
+      label="Assign scope"
+      i18n-label
+      [errorMessage]="formFieldErrors()('scopeValue')"
+    >
+      <input
+        pInputText
+        type="text"
+        formControlName="scopeValue"
+        autocomplete="off"
+        i18n-placeholder
+        class="mb-2 font-mono"
```

Comment:
```suggestion
        class="font-mono"
```

Same as the other margin comments above